src/ng/browser.js

```diff
@@ -67,7 +67,7 @@ function Browser(window, document, $log, $sniffer) {
 
   /**
    * @private
-   * Note: this method is used only by scenario runner
+   * Note: this method is used only by protractor / scenario runner
```

Comment:
Is protractor really using this? I think it uses testability.

Reply:
Testability.whenStable calls this function

Reply:
I know, but this comment does not belong here. For example, we have discussed in the past that we should make Testability public, which means that anyone could use `whenStable()`. And if we do that, it is very unlikely that someone will remember to update this comment.

If we want to have such a comment, it should be on `whenStable()` imo.